commands.go

```diff
@@ -214,6 +214,30 @@ func toBeFairResponse(ctx context.Context) string {
 	return pickGif
 }
 
+func toBeFairAutoResponse(s *discordgo.Session, m *discordgo.MessageCreate) {
+	ctx := context.Background()
+	var span *trace.Span
+	me := hnydiscordgo.MessageEvent{Message: m.Message, Context: ctx}
+
+	ctx, span = hnydiscordgo.StartSpanOrTraceFromMessage(&me, s)
+
+	span.AddField("command", "toBeFairAutoResponse")
+
+	deciders := []string{"no", "yes"}
+	decisionResponse, randNum := chooseRandom(deciders)
+	span.AddField("toBeFairAutoResponse.randomNumber", randNum)
+
+	if decisionResponse == "yes" {
+		span.AddField("toBeFairAutoResponse.RandomDecision", "Yes, responding")

```

Comment:
Can we swap this and the other to `true` and `false`? Just makes querying things on Honeycomb a bit more intuitive

Reply:
As in the result coming from the random generator? I'd need to make it more generic (currently it takes and returns strings explicitly.)

Could always translate the string into bool representations once they're back?

Reply:
I was thinking of just changing `"yes, responding"` to `true` as it'll be more searchable.

Reply:
Ooh, right, gotcha